src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -496,7 +496,7 @@ public async Task SendNewDeviceLoggedInEmail(string email, string deviceType, Da
             SiteName = _globalSettings.SiteName,
             DeviceType = deviceType,
             TheDate = timestamp.ToLongDateString(),
-            TheTime = timestamp.ToShortTimeString(),
+            TheTime = timestamp.ToString("hh:mm:ss.fff tt"),
```

Comment:
My first instinct was to try to just have second precision: `hh:mm:ss tt` as I think that walks the line between human readable and searchable for logs.